exporter/splunkhecexporter/config.go

```diff
@@ -117,6 +138,12 @@ type Config struct {
 
 	// UseMultiMetricFormat combines metric events to save space during ingestion.
 	UseMultiMetricFormat bool `mapstructure:"use_multi_metric_format"`
+
+	// HecHeartbeat is the configuration to enable heartbeat
+	HecHeartbeat HecHeartbeat `mapstructure:"hec_heartbeat"`
+
+	// HecTelemetry is the configuration for splunk hec exporter telemetry
+	HecTelemetry HecTelemetry `mapstructure:"telemetry"`
```

Comment:
Can you please call the fields the same as the `yaml` representation?
```suggestion
	Telemetry HecTelemetry `mapstructure:"telemetry"`
```

We are trying to follow this pattern across the collector's codebase. The HEC exporter is not a good example tho :)

Reply:
fix both the telemetry and heartbeat.